### DISCUSSION TITLE
osu! lazer cant import skins on Linux

### DISCUSSION THREAD

**Discussion by criptixo** [ppy/osu#15004] [Q&A] [answered]
Im using the latest osu! lazer appimage on my arch linux system, i want to import a skin but the the "select a directory" button is grayed out. (I don't have osu! stable installed just a .osk skin)
is this a bug ?

**Comment by zzahkaboom24** [comment, marked as answer]
I am gonna try to answer this too even tho i have no knowledge with linux. Have you tried dragging the osk file into the osu!lazer window?

**Reply by criptixo** [reply]
that worked, thanks
